### PR TITLE
PXB-3020 Release tasks for PXB-8.0.32-26

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh [hunk 1]
```diff
@@ -243,9 +243,9 @@ install_deps() {
             PKGLIST+=" binutils-devel python3-pip python3-setuptools"
             PKGLIST+=" libcurl-devel cmake libaio-devel zlib-devel libev-devel bison make gcc"
             PKGLIST+=" rpm-build libgcrypt-devel ncurses-devel readline-devel openssl-devel gcc-c++"
-            PKGLIST+=" vim-common rpmlint patchelf python3-sphinx python3-wheel"
+            PKGLIST+=" vim-common rpmlint patchelf python3-wheel"
```
Comment (satya-bodapati) [inline storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh]: may I know why this is changed?
We require sphynix only on RHEL 9/OEL9?

**Review by altmannmarcelo — APPROVED** [review]
LGTM

File: storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh (hunk 1 above)
Comment (vorsel) [reply storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh]: Hi @satya-bodapati 
This change is OK for the build script.

**Review by vorsel — APPROVED** [review]

File: storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh (hunk 1 above)
Comment (satya-bodapati) [reply storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh]: Ok

**Pull request merged**